defaults-pwgmmtest.sh

```diff
@@ -5,15 +5,57 @@ env:
   CFLAGS: "-fPIC -g -O2"
   CMAKE_BUILD_TYPE: "RELWITHDEBINFO"
 overrides:
+# taken from defaults-root6
+  GCC-Toolchain:
+    tag: v6.2.0-alice1
+    prefer_system_check: |
+      set -e
+      which gfortran || { echo "gfortran missing"; exit 1; }
+      which cc && test -f $(dirname $(which cc))/c++ && printf "#define GCCVER ((__GNUC__ << 16)+(__GNUC_MINOR__ << 8)+(__GNUC_PATCHLEVEL__))\n#if (GCCVER < 0x060200)\n#error \"System's GCC cannot be used: we need at least GCC 6.X. We are going to compile our own version.\"\n#endif\n" | cc -xc++ - -c -o /dev/null
+  ROOT:
+    version: "%(tag_basename)s"
+    tag: "v6-10-08"
+    source: https://github.com/root-mirror/root
+    requires:
+      - AliEn-Runtime:(?!.*ppc64)
+      - GSL
+      - opengl:(?!osx)
+      - Xdevel:(?!osx)
+      - FreeType:(?!osx)
+      - Python-modules
+      - "GCC-Toolchain:(?!osx)"
+      - libpng
+      - lzma
+  GSL:
+    prefer_system_check: |
+      printf "#include \"gsl/gsl_version.h\"\n#define GSL_V GSL_MAJOR_VERSION * 100 + GSL_MINOR_VERSION\n# if (GSL_V < 116)\n#error \"Cannot use system's gsl. Notice we only support versions from 1.16 (included)\"\n#endif\nint main(){}" | gcc  -I$(brew --prefix gsl)/include -xc++ - -o /dev/null
+  AliRoot:
+    version: "%(commit_hash)s_PWGMMTEST"
+    tag: v5-09-22
+  AliPhysics:
+    version: "%(commit_hash)s_PWGMMTEST"
+    tag: v5-09-22-01
+# now for actual generators
+  agile:
```

Comment:
Can you add AliGenerators too? We need the `_PWGMMTEST` suffix for that package (see alisw/ali-bot#540).

Reply:
Sure, done.